101-automation-configuration/nested/provisionKeyVault.json

```diff
@@ -0,0 +1,119 @@
+      "location": {
+        "type"        : "string",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata"    : {
+          "description": "Specifies the Azure location where the key vault should be created."
+        }
+      },
+      "enabledForDeployment": {
+        "type"         : "bool",
+        "defaultValue" : false,
+        "allowedValues": [
+          true,
+          false
+        ],
+        "metadata": {
+          "description": "Specifies whether Azure Virtual Machines are permitted to retrieve certificates stored as secrets from the key vault."
+        }
+      },
+      "enabledForDiskEncryption": {
+        "type"         : "bool",
+        "defaultValue" : false,
+        "allowedValues": [
+          true,
+          false
+        ],
+        "metadata": {
+          "description": "Specifies whether Azure Disk Encryption is permitted to retrieve secrets from the vault and unwrap keys."
+        }
+      },
+      "enabledForTemplateDeployment": {
+        "type"         : "bool",
+        "defaultValue" : false,
+        "allowedValues": [
+          true,
+          false
+        ],
+        "metadata": {
+          "description": "Specifies whether Azure Resource Manager is permitted to retrieve secrets from the key vault."
+        }
+      },
+      "tenantId": {
+        "type"        : "string",
+        "defaultValue": "[subscription().tenantId]",
+        "metadata"    : {
+          "description": "Specifies the Azure Active Directory tenant ID that should be used for authenticating requests to the key vault. Get it by using Get-AzSubscription cmdlet."
+        }
+      },
+      "skuName": {
+        "type"         : "string",
+        "defaultValue" : "Standard",
+        "allowedValues": [
+          "Standard",
+          "Premium"
+        ],
+        "metadata": {
+          "description": "Specifies whether the key vault is a standard vault or a premium vault."
+        }
+      },
+      "secretName": {
+        "type"    : "string",
+        "metadata": {
+          "description": "Specifies the name of the secret that you want to create."
+        }
+      },
+      "secretValue": {
+        "type"    : "securestring",
+        "metadata": {
+          "description": "Specifies the value of the secret that you want to create."
+        }
+      }
+    },
+    "resources": [
+      {
+        "type"      : "Microsoft.KeyVault/vaults",
+        "name"      : "[parameters('keyVaultName')]",
+        "apiVersion": "2018-02-14",
+        "location"  : "[parameters('location')]",
+        "properties": {
+          "enabledForDeployment"        : "[parameters('enabledForDeployment')]",
+          "enabledForDiskEncryption"    : "[parameters('enabledForDiskEncryption')]",
+          "enabledForTemplateDeployment": "[parameters('enabledForTemplateDeployment')]",
+          "tenantId"                    : "[parameters('tenantId')]",
+          "accessPolicies"              : [],
```

Comment:
remove empty properties

Reply:
https://github.com/Azure/azure-quickstart-templates/issues/5517

Errors if you remove it.